### DISCUSSION TITLE
How to use fods for fiber tracking

### DISCUSSION THREAD

**Discussion by cxmDREAM** [dipy/dipy#3313] [Reconstruction] [answered]
Hello, developer
I used the Watson model from the DMIPY library to generate fods, as shown in the following code

- **NODDI_mod = MultiCompartmentModel(models=[ball, watson_dispersed_bundle])
- NODDI_fit_hcp = NODDI_mod.fit(scheme_hcp, data_hcp, mask=data_hcp[..., 0]>0)
- fods = NODDI_fit_hcp.fod(sphere.vertices, visual_odi_lower_bound=0.08)[70:90,: , 70:90]**

I now want to use the LocalTracking function in dipy to track fibers. The effect is as shown in this example,
https://docs.dipy.org/stable/examples_built/visualization/viz_roi_contour.html#
but I am confused about how to convert fods into director_getter parameters. Is there any other way to generate a streamline? I hope to receive your guidance, thank you

**Comment by skoudoro** [comment, marked as answer]
Hi @cxmDREAM,

you should look at this tutorial: https://docs.dipy.org/stable/examples_built/fiber_tracking/tracking_probabilistic.html

Especially `ProbabilisticDirectionGetter.from_shcoeff`.  It should help for your tracking.

**Reply by cxmDREAM** [reply]
Thank you, your answer is very useful to me
